azext_iot/commands.py

```diff
@@ -5,11 +5,13 @@
 # --------------------------------------------------------------------------------------------
 
 from azure.cli.core.commands import CliCommandType
+from azext_iot._factory import iot_hub_service_factory 
+from azext_iot._factory import iot_service_provisioning_factory
 from azext_iot import iotext_custom
 
 
 def load_command_table(self, _):
-    with self.command_group('iot hub') as g:
+    with self.command_group('iot hub', client_factory=iot_hub_service_factory) as g:
```

Comment:
Because iot_hub_service_factory is the default client_factory, you don't need to set it here...DPS commands should override though.

Reply:
sure. will remove